.travis.yml

```diff
@@ -1,19 +1,26 @@
 ---
+sudo: required
 language: python
-python:
-  - "3.6"
-
 services:
   - docker
-
+before_install:
+  - sudo apt-get -qq update
+env:
+  - molecule_distro=centos7
+  # - molecule_distro=centos8
+  # - molecule_distro=debian8
+  # - molecule_distro=debian9
+  # - molecule_distro=debian10
+  # - molecule_distro=fedora
+  - molecule_distro=ubuntu1604
+  - molecule_distro=ubuntu1804
+  # - molecule_distro=ubuntu2004
```

Comment:
Is focal not being tested? I see molecule files for them?

Reply:
I believe they were failing for some reason. I need to review again.

Reply:
https://github.com/mrlesmithjr/ansible-frr/runs/971531291?check_suite_focus=true#step:5:131

Reply:
Okay let's get this merged for now - I'll look at some of the testing later on 👍

Reply:
Merging now!